django-stubs/contrib/admin/views/main.pyi

```diff
@@ -37,8 +41,13 @@ class ChangeList:
     page_num: int
     show_all: bool
     is_popup: bool
+    add_facets: bool
+    is_facets_optional: bool
     to_field: Any
     params: dict[str, Any]
+    filter_params: dict[str, Any]
```

Comment:
I think this is

```suggestion
    filter_params: dict[str, list[str]]
```

Reply:
Thanks! It was fixed at https://github.com/typeddjango/django-stubs/pull/2079/commits/7885d68c3acb7d8752b869ca45021d986b2859b7.